de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/modelaccess/CategoryAssignmentResource.java

```diff
@@ -114,6 +122,58 @@ public Response putCa(@ApiParam(value = "CA to put", required = true) ABeanCateg
 		}
 	}
 	
+	/** **/
+	@POST
+	@Path("/{parentUuid}")
+	@Consumes(MediaType.APPLICATION_JSON)
+	@ApiOperation(
+			consumes = "application/json",
+			value = "Create CA",
+			httpMethod = "POST",
+			notes = "This service creates a new CategoryAssignment and returns it")
+	@ApiResponses(value = { 
+			@ApiResponse(
+					code = HttpStatus.OK_200,
+					response = String.class,
+					message = ApiErrorHelper.SUCCESSFUL_OPERATION),
+			@ApiResponse(
+					code = HttpStatus.BAD_REQUEST_400, 
+					message = ApiErrorHelper.COULD_NOT_FIND_REQUESTED_ELEMENT),
+			@ApiResponse(
+					code = HttpStatus.INTERNAL_SERVER_ERROR_500, 
+					message = ApiErrorHelper.SYNC_ERROR)})
+	public Response createCa(@PathParam("parentUuid") @ApiParam(value = "parent uuid", required = true) String parentUuid,
+			@QueryParam(value = ModelAccessResource.QP_FULL_QUALIFIED_NAME)
+			@ApiParam(value = "Full qualified name of the CA type", required = true) String fullQualifiedName) {
+		try {
+			serverRepository.syncRepository();
+			
+			StructuralElementInstance parentSei = RepositoryUtility.findSei(parentUuid, repository);
+			
+			if (parentSei == null) {
+				return ApiErrorHelper.createNotFoundErrorResponse();
+			}
+			
+			ActiveConceptHelper helper = new ActiveConceptHelper(ed.getResourceSet().getRepository());
+			
+			CategoryAssignment newCa = CategoriesFactory.eINSTANCE.createCategoryAssignment();
```

Comment:
Maybe a minor simplification is possible using the CategoryInstantiator. What is the current default name of the instantiated category assignment? Null?